backend/src/main/java/com/woowacourse/levellog/team/exception/InterviewTimeException.java

```diff
@@ -0,0 +1,20 @@
+package com.woowacourse.levellog.team.exception;
+
+import com.woowacourse.levellog.common.exception.LevellogException;
+import org.springframework.http.HttpStatus;
+
+/**
+ * 요청이 들어온 시간과 인터뷰 시작 시간을 비교했을 때 정책과 맞지 않을 경우 발생하는 예외
+ */
+public class InterviewTimeException extends LevellogException {
+
+    private static final String CLIENT_MESSAGE = "잘못된 인터뷰 시작 시간입니다.";
+
+    public InterviewTimeException(final String exceptionInfoForDev, final String clientMessage) {
+        super(clientMessage + exceptionInfoForDev, clientMessage, HttpStatus.BAD_REQUEST);
+    }
```

Comment:
기존에는 `message + 특정 값` 이런식으로 표현했던 걸 기반으로 한다면 아래와 같이 메서드 순서를 바꿔주는 것이 좋을 것 같아요 😉
```suggestion
    public InterviewTimeException(final String clientMessage, final String exceptionInfoForDev) {
        super(clientMessage + exceptionInfoForDev, clientMessage, HttpStatus.BAD_REQUEST);
    }
```

- as is
```java
throw new InterviewTimeException("[teamId : " + this.getId() + "]", "인터뷰가 시작되기 전에 종료할 수 없습니다.");
```
- to be
```java
throw new InterviewTimeException("인터뷰가 시작되기 전에 종료할 수 없습니다.", "[teamId : " + this.getId() + "]");
```